### DISCUSSION TITLE
获取服务对象

### DISCUSSION THREAD

**Discussion by 10-Hun-01** [openTCS/opentcs#153] [Q&A]
KernelServicePortal servicePortal =  new KernelServicePortalBuilder("userName","password").build();
servicePortal.login("127.0.0.1", 1099);

异常信息： Authentication failed for user userName

**Comment by swltr** [comment]
@10-Hun-01 Out of the box, you can just use `org.opentcs.common.GuestUserCredentials.USER` and `org.opentcs.common.GuestUserCredentials.PASSWORD`.

Alternatively, you'd need to bind `org.opentcs.kernel.extensions.rmi.UserAccountProvider` to your own implementation that actually provides user accounts other than the default one.
